Rovers/Rovers.tex

```diff
@@ -0,0 +1,387 @@
+Though a rover can be made without them additional funding allows for the rover project to expand in scope. 
+We are aiming for the \$2500.00 price point to achieve the MVP as well as improvements like extended battery, Lidar, communications and improved navigation. There are some improvements that can be made under the \$500.00 price point including the advanced navigation station and partial autonomy. 
+However, it is noteworthy that the project is flexible enough to function on various amounts of funding as long as it is above the \$500.00 price point.
+
+\subsection{Faculty Support}
+\label{subsec:faculty}
+Support from faculty could greatly advance this study and what RIT SPEX is capable robotics wise. 
+It is not necessary but would be very helpful. 
+Professors could help with funding, rover design, hardware selection and general advice. 
+Rovers and robotics are new to SPEX and someone with experience could help this. 
+The team should reach out to professors for advice and help. 
+There are many professors at RIT with interest in robotics or computer vision. 
+With some recommendations from SPEX alumni we have a short list of professors who might be interested. 
+There is also a large robotics competition which is hosted yearly at RIT, FIRST robotics. 
+This is also a potential source of support from RIT faculty. 
+
+\begin{table}[ht!]
+    \caption{Potential Faculty Support}
+    \centering
+    {\renewcommand{\arraystretch}{1.5}
+    \begin{tabularx}{\linewidth}{LLL} 
+    \hline
+    \textbf{Professor} & \textbf{Department / Field} & \textbf{Email Address} \\
+    \hline
+    Dr. Ugur Sahin & Mathematics (Researches Computer Vision) & us-grd@cs.rit.edu \\
+    Dr Ferat Sahin & KGCOE (Robotics) & feseee@rit.edu \\
+    \hline
+    \end{tabularx}
+    }
+\label{tab:fac-sup}
+\end{table}
+
+\subsection{Long-Term Vision}
+\label{subsec:vision}
+The long-term vision of this project is to open RIT Space Exploration up to a new area of projects and development. 
+Robotics and rovers are at the core of deep space exploration and most science missions. 
+The project is also intended to lead into a future rover project with expanded capabilities.
+The University Rover Challenge is hosted by the Mars Society. 
+It features very competitive and expensive rovers. 
+These rovers feature autonomous navigation, fine control robotics, scientific drill and more. 
+It is a worthy long term goal. 
+
+\section*{Acknowledgements}
+The author would like to thank SPEX alumni Phil Linden for creating the PDD template, SPEX alumni 'TJ' Tarazevits for feedback on project design and management, SPEX alumni Daniel Mitchell and Matthew Glazer for their feedback on electrical system design design, Anthony Hennig for founding RIT Space Exploration, and all the SPEX members that continue to invest their time and energy into the pursuit of space exploration.
+
```

Comment:
The URC rulebook or Standard Operating Procedures would make a great reference to include.